discourse_forum/discourse_forum/spiders/discourse_spider.py

```diff
@@ -119,6 +124,28 @@ def extract_core_from_post(post):
             'position': post_position}
 
 
+def extract_block_quotes_from_post(post):
+    quotes = post.css('aside.quote')
+    output = []
+    for quote in quotes:
+        quote_content = quote.css('blockquote p::text').get()
+        data_username = quote.attrib.get('data-username')
+        data_post = quote.attrib.get('data-post')
```

Comment:
i dont understand what is the `data-post` and `data-topic`

Reply:
The `data-post` means the content/data in the post being quoted. The `data-topic` the information related to the topic of the post, as an id. I will remove these attributes, as of now, to keep the graph structure as simple as possible.